src/Classes/TradeQuery.lua

```diff
@@ -212,61 +126,50 @@ end
 
 -- Method to pull down and interpret the PoE.Ninja JSON endpoint data
 function TradeQueryClass:PullPoENinjaCurrencyConversion(league, controls)
+	local now = os.time()
```

Comment:
This should probably be using `get_time()` since we're using it elsewhere anyway